src-ghc-tree/Internal.hs

```diff
@@ -1,13 +1,14 @@
-{-# LANGUAGE RecordWildCards #-}
-{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE CPP #-}
```

Comment:
With ghc-9.4 dropping derived constraints could we split the source again and avoid CPP use?